### PR TITLE
PoS-Plasma using metamask

### PR DESCRIPTION
**Pull request by abhishektanwar**

### CODE REVIEW

**Comment by rahuldamodar94** [comment]
keep the name of the .md file in smaller case. 
add the tutorial properly to the sidebar.js file.

**Review by rahuldamodar94 — CHANGES_REQUESTED** [review]
Run the docs in your local and go through it once before raising PR. 
This doc has not been included in the sidebar.

File: docs/develop/pos-plasma-using-metamask.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+description: PoS-Plasma token transfer tutorial using metamask.
+keywords:
+  - docs
+  - matic
+image: https://matic.network/banners/matic-network-16x9.png 
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+This tutorial is a brief introduction on how to transfer tokens between Ethereum and Matic using PoS or Plasma using matic.js library and Metamask. Matic-Ethereum bridge provides a cross-chain channel using which users can transfer tokens from Ethereum to Matic and vice-versa. Both PoS and Plasma bridge is available to the user to transfer the tokens across two chains. More details on using the bridge can be found [here](https://docs.matic.network/docs/develop/ethereum-matic/getting-started). This tutorial mainly focuses on using the bridge from a front end perspective. We will be using Metamask for this purpose.
```
Comment (rahuldamodar94) [inline docs/develop/pos-plasma-using-metamask.md:13]: change the current sentence to

how to transfer tokens between Ethereum and Matic on PoS and Plasma bridge using matic.js library and Metamask

Comment (rahuldamodar94) [reply docs/develop/pos-plasma-using-metamask.md:13]: also please refer other docs and figure out how to properly give links to other pages in the docs

File: docs/develop/pos-plasma-using-metamask.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- plasmaWETH : Plasma WETH
+- plasmaRootERC20 : ERC20 root token deployed on plasma
+- plasmaChildERC20 : ERC20 child token deployed on plasma
+
+4. Run the project using `npm start` .
+## Example using PoS ERC20
+
+### Deposit
+
+In deposit functionality for ERC20 tokens first, approval is given then deposit takes place. Upon clicking the deposit button metamask will first ask to approve the transfer of a specified number of tokens and after the confirmation of the approval transaction, metamask will ask to confirm the deposit transaction. Make sure the ethereum network is selected in metamask for deposit functionality.
```
Comment (rahuldamodar94) [inline docs/develop/pos-plasma-using-metamask.md:34]: change this to 

To deposit ERC20 tokens, an approve function call has to be made before calling the deposit function. Upon clicking the deposit button metamask will first asks to approve the transfer of a specified number of tokens and after the confirmation of the approval transaction, metamask will ask to confirm the deposit transaction. Make sure the goerli network is selected in metamask for deposit functionality.

File: docs/develop/pos-plasma-using-metamask.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+During deposit of ERC20 tokens providers are specified as below
```
Comment (rahuldamodar94) [inline docs/develop/pos-plasma-using-metamask.md:36]: During deposit of ERC20 tokens, the providers are specified as below

File: docs/develop/pos-plasma-using-metamask.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+<img src={useBaseUrl("img/pos-plasma-using-metamask/approve.png")} />
+
+<img src={useBaseUrl("img/pos-plasma-using-metamask/deposit.png")} />
+
+
+
+### Burn
+
+Burning of tokens takes place on Matic, make sure Matic network is selected in metamask.
```
Comment (rahuldamodar94) [inline docs/develop/pos-plasma-using-metamask.md:50]: For withdrawing tokens back to root chain,tokens have to be first burnt on child chain, make sure Mumbai network is selected in metamask.

File: docs/develop/pos-plasma-using-metamask.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+During burning of ERC20 tokens providers are specified as below
```
Comment (rahuldamodar94) [inline docs/develop/pos-plasma-using-metamask.md:52]: For burning the ERC20 tokens, providers are specified as below

File: docs/develop/pos-plasma-using-metamask.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+`maticProvider: window.web3`
+
+`parentProvider: ethereumprovider`
+
+<img src={useBaseUrl("img/pos-plasma-using-metamask/burn.png")} />
+
+### Exit
+
+The exit process takes place on ethereum and upon confirmation on ethereum, equivalent amount tokens burn on Matic are released on ethereum. Make sure the ethereum network is selected in metamask. The burn hash obtained after burning of tokens is given as an argument. Wait for the checkpointing to complete before doing exit process. Checkpoint time is usually 5 minutes.
```
Comment (rahuldamodar94) [inline docs/develop/pos-plasma-using-metamask.md:62]: The exit process takes place on ethereum and upon confirmation, equivalent amount of tokens that were burnt on ch ild chain are released to the users address on root chain. Make sure the Goerli network is selected in metamask. The burn hash obtained after burning of tokens is given as the input. Wait for the checkpointing to complete before doing this exit process. Checkpoint time is usually ~10 minutes.

File: docs/develop/pos-plasma-using-metamask.md (hunk 1 above)
Comment (abhishektanwar) [reply docs/develop/pos-plasma-using-metamask.md]: fixed

File: docs/develop/pos-plasma-using-metamask.md (hunk 3 above)
Comment (abhishektanwar) [reply docs/develop/pos-plasma-using-metamask.md]: fixed

File: docs/develop/pos-plasma-using-metamask.md (hunk 5 above)
Comment (abhishektanwar) [reply docs/develop/pos-plasma-using-metamask.md:52]: fixed

File: docs/develop/pos-plasma-using-metamask.md (hunk 6 above)
Comment (abhishektanwar) [reply docs/develop/pos-plasma-using-metamask.md]: fixed

**Review by rahuldamodar94 — APPROVED** [review]

**Pull request merged**